core/services/job/models.go

```diff
@@ -214,9 +214,11 @@ type OffchainReportingOracleSpec struct {
 	ContractConfigTrackerPollIntervalEnv      bool
 	ContractConfigConfirmations               uint16 `toml:"contractConfigConfirmations"`
 	ContractConfigConfirmationsEnv            bool
-	EVMChainID                                *utils.Big `toml:"evmChainID" db:"evm_chain_id"`
-	CreatedAt                                 time.Time  `toml:"-"`
-	UpdatedAt                                 time.Time  `toml:"-"`
+	EVMChainID                                *utils.Big      `toml:"evmChainID" db:"evm_chain_id"`
+	DatabaseTimeout                           models.Interval `toml:"databaseTimeout"`
```

Comment:
we'll need an associated db migration and an update to the CreateJob function

Reply:
Done